schema/cassandra/cadence/schema.cql

```diff
@@ -427,6 +431,7 @@ CREATE TABLE domains (
   };
 
 CREATE TABLE domains_by_name_v2 (
+  created_time                  timestamp,
```

Comment:
mind adding an updated time too?

Reply:
I don't get it why we are adding last_updated_time when it is immutable?

Reply:
I don't think this table is immutable, I think you're referring to the `domains` table? and in that case it's not really immutable as much as we're just not changing it right now (need to finish that migration I guess)

Reply:
oh! For domains_by_name_v2 it has last_updated_time in bigint already

Reply:
ah ok, it's a subfield, that I guess works